### PR TITLE
Separate models, processors and helpers

### PR DESCRIPTION
**Pull request by niktrix**

### CODE REVIEW

**Comment by niktrix** [comment]
#58

File: apps/ethvm/src/libs/index.ts [hunk 1]
```diff
@@ -1,18 +1,13 @@
...
 import Hash from '@app/libs/Hash'
 import Hex from '@app/libs/Hex'
 import HexNumber from '@app/libs/HexNumber'
 import HexTime from '@app/libs/HexTime'
 
 import FIFO from '@app/libs/FIFO'
-import Tx from '@app/libs/Tx'
-import processTxs from '@app/libs/txProcessor'
 
 import ethUnits from 'ethereumjs-units'
```
Comment (aldoborrero) [inline apps/ethvm/src/libs/index.ts]: Remove this import from here is not being used perse (in other srcs it's being imported directly)

File: apps/ethvm/src/processor/index.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import processTxs from '@app/processor/txProcessor'
```
Comment (aldoborrero) [inline apps/ethvm/src/processor/index.ts]: Remove this lines and use export * from '@app/processor/txProcessor' syntax

File: apps/ethvm/src/libs/index.ts (hunk 1 above)
Comment (niktrix) [reply apps/ethvm/src/libs/index.ts]: Done

File: apps/ethvm/src/processor/index.ts (hunk 2 above)
Comment (niktrix) [reply apps/ethvm/src/processor/index.ts]: done

**Pull request merged**